src/routes/admin/user.js

```diff
@@ -137,11 +174,42 @@ export async function updateUser(req, res) {
         individualHooks: true,
       },
     );
-    await Permission.destroy({ where: { userId } });
-    await Permission.bulkCreate(requestUser.permissions, { validate: true, individualHooks: true });
 
-    // User roles are handled a bit more clumsily
-    await UserRole.destroy({ where: { userId } });
+    const ALL_REGIONS = 15;
```

Comment:
Thank you, we have needed this for a long time, this addresses. I think this also fixes TTAHUB-670